roles/girder-install/tasks/main.yml

```diff
@@ -1,45 +1,28 @@
 ---
-
-  - name: apt | cache | update
-    apt: update_cache=yes
-    when: do_install|bool
+  - name: girder | add | node repository
+    shell: "curl -sL https://deb.nodesource.com/setup | sudo bash -"
```

Comment:
lets talk about how you would like to see that implemented.  Building from source is extremely time consuming.